### DISCUSSION TITLE
Why decreasing of node pool min_count decreases cost estimate?

### DISCUSSION THREAD

**Discussion by mloskot** [infracost/infracost#2368] [Q&A]
I decreased `min_count=1` of my AKS node pool resource to `min_count=1`, so the new resources looks like this

```hcl
resource "azurerm_kubernetes_cluster_node_pool" "node_pools" {
  name             = "npw1ph"
  node_count       = 1
  min_count        = 0
  max_count        = 3
  ...
}
```

Infracost calculated new estimate as very much decreased in comparison to the previous estimate for `min_count=1`

```
~ module.aks_windows_node_pool.azurerm_kubernetes_cluster_node_pool.node_pools[1]
  -£98.71 (£98.71 → £0.00)
    ~ Instance usage (pay as you go, Standard_E2as_v5)
      -£79.25 (£79.25 → £0.00)
    ~ os_disk
        ~ Storage (P10, LRS)
          -£19.47 (£19.47 → £0.00)
```

Why decreasing of `min_count` affects the cost estimate?
Shouldn't `node_count=1` prevent it?

In https://github.com/infracost/infracost/pull/2136, in code comment, I'm reading  this

> if the node count is not set explicitly let's take the min_count.

Why `min_count` is taken into account in my case despite that I do set the `node_count`?

**Comment by vdmgolub** [comment]
Hello! Thank you for asking this!

This looks like a bug. I've created a new issue to track it: https://github.com/infracost/infracost/issues/2371. Going to close this discussion to move over to the issue.

Thanks again for highlighting this! :)

**Reply by mloskot** [reply]
Thanks
